plugins/inputs/cisco_telemetry_mdt/sample.conf

```
@@ -21,6 +21,9 @@
  ## Define (for certain nested telemetry measurements with embedded tags) which fields are tags
  # embedded_tags = ["Cisco-IOS-XR-qos-ma-oper:qos/interface-table/interface/input/service-policy-names/service-policy-instance/statistics/class-stats/class-name"]
 
+## Include the delete field in every telemetry message.
+  include_delete_field = true
+
```

This is not correct. The default should keep backward compatibility. Furthermore, this is not an option the user _has to_ set so I'd rather like to see this as a (commented) optional param with the default:
```suggestion
  ## Include the delete field in every telemetry message.
  # include_delete_field = false
```